drivers/gpio/gpiolib-acpi.c

```diff
@@ -1599,6 +1599,33 @@ static const struct dmi_system_id gpiolib_acpi_quirks[] __initconst = {
 			.ignore_interrupt = "AMDI0030:00@18",
 		},
 	},
+	{
+		/*
+		 * Spurious wakeups from TP_ATTN# pin
+		 * Found in BIOS 1.7.8
+		 * https://gitlab.freedesktop.org/drm/amd/-/issues/1722#note_1720627
+		 */
+		.matches = {
+			DMI_MATCH(DMI_BOARD_NAME, "NL5xRU"),
+		},
+		.driver_data = &(struct acpi_gpiolib_dmi_quirk) {
+			.ignore_wake = "ELAN0415:00@9",
+		},
+	},
+	{
+		/*
+		 * Spurious wakeups from TP_ATTN# pin
+		 * Leveraged HW design from Clevo NL50NU
+		 * https://bugzilla.redhat.com/show_bug.cgi?id=2162013
+		 */
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "System76"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Pangolin"),
```

Comment:
@13r0ck this over-matches, use product version, one for pang10 and another for pang11, or find the latest version of the patch.